recipes/icu/all/conanfile.py

```diff
@@ -1,19 +1,26 @@
 from conan import ConanFile
 from conan.tools.apple import is_apple_os
-from conan.tools.build import cross_building
+from conan.tools.build import cross_building, stdcpp_library
 from conan.tools.env import Environment, VirtualBuildEnv
 from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, mkdir, rename, replace_in_file, rm, rmdir, save
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
 from conan.tools.scm import Version
-from conans.tools import get_gnu_triplet, sha256sum, stdcpp_library
+from conan.errors import ConanInvalidConfiguration
 import glob
 import os
 import shutil
+import hashlib
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=1.54.0"
 
+def _sha256sum(file_path):
```

Comment:
copying an implementation to every recipe doesn't scale, it has to be exposed in 2.0
https://github.com/conan-io/conan/issues/13073

Reply:
Only this recipe and another test package are using sha256sum, I'm not sure if we really need to port it Conan v2, I mean, only few usage, and still, it could even more simplified to `hashlib.sha256(load(self, file_path).encode('UTF-8')).hexdigest()`